Source/WebCore/editing/Editing.cpp

```diff
@@ -784,9 +784,9 @@ bool isEmptyTableCell(const Node* node)
 Ref<HTMLElement> createDefaultParagraphElement(Document& document)
 {
     switch (document.editor().defaultParagraphSeparator()) {
-    case EditorParagraphSeparatorIsDiv:
+    case EditorParagraphSeparator::div:
         return HTMLDivElement::create(document);
-    case EditorParagraphSeparatorIsP:
+    case EditorParagraphSeparator::p:
```

Comment:
Is it intentional that the enum values are `div` and `p` instead of `Div` and `P`/`Paragraph`? (Enum values are capitalized for the most part, but I suppose we may want to match the tag names exactly here).

Reply:
Yeah, I intentionally made them lowercase to be consistent with tag names.